src/org/javarosa/xform/parse/FormInstanceParser.java

```diff
@@ -0,0 +1,629 @@
+package org.javarosa.xform.parse;
+
+import org.javarosa.core.model.*;
```

Comment:
In general I agree with that (and certainly in this case). Not cluttering the namespace, showing exactly what’s used, etc. An exception is when the type contains something like constants, and they all are used. My view anyway.